site/content/en/docs/tasks/dev/integrate_a_custom_job.md

```diff
@@ -93,22 +93,13 @@ Here are completed external integrations you can learn from:
 Add your framework's GroupVersionKind to `.integrations.externalFrameworks` in [controller_manager_config.yaml](
```

Comment:
Maybe you can add an extra sentence in line 89 along the lines of "Building an external integration is similar to building a built-in integration, except for ..."

Reply:
Done